manual/cli.rst

```diff
@@ -1,4 +1,100 @@
++===========+============================+======================================================================================================+
+| -r FILE   | --rivers=FILE              | produce a map of rivers, after the option it expects the name of the file where to generate the map  |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| --gs      | --grayscale-heightmap      | produce a grayscale heightmap                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --ocean_level=N            |  elevation cut off for sea level [default = 1.0]                                                     |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --temps #/#/#/#/#/#        |  Provide alternate ranges for temperatures. [default = .126/.235/.406/.561/.634/.876]                |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --humidity #/#/#/#/#/#/#   | Provide alternate ranges for humidities. [default = .059/.222/.493/.764/.927/.986/.998]              |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -gv N     | --gamma-value N            | N = Gamma value for temperature/precipitation gamma correction curve. [default = 1.25]               |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -go N     | --gamma-offset N           | N = Adjustment value for temperature/precipitation gamma correction curve. [default = .2]            |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | ---not-fade-borders        | avoid fading borders                                                                                 |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --scatter                  | generate scatter plot                                                                                |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --sat                      | generate satellite map                                                                               |
```

Comment:
If there is a chance that https://github.com/Mindwerks/worldengine/pull/197 will be merged, this could be removed. The satellite-map is too pretty to be hidden behind a paramter.